saml-core/src/main/java/org/keycloak/saml/processing/core/util/IDFedLSInputResolver.java

```diff
@@ -49,6 +49,7 @@
         // XML Schema/DTD
         schemaLocationMap.put("datatypes.dtd", "schema/w3c/xmlschema/datatypes.dtd");
         schemaLocationMap.put("XMLSchema.dtd", "schema/w3c/xmlschema/XMLSchema.dtd");
+        schemaLocationMap.put("http://www.w3.org/2001/XMLSchema.dtd", "schema/w3c/xmlschema/XMLSchema.dtd");        
```

Comment:
The previous PR changed the SPMetadataDescriptor which generates XML. So the unit test ValidationTest was updated to actually schema-validate the generated output. However this did not work right without a few fixes to IDFedLSInputResolver. Constructing a dedicated test for the resolver is quite difficult, since it need to interfere with outgoing http connections. The best way is to just disconnect the internet and see that schema loading still works. Any ideas?